src/Void.ViewModel/RenderWindows.fs

```diff
@@ -26,33 +26,83 @@ module RenderWindows =
             let lineNotInBufferAsDrawingObject i =
                 DrawingObject.Text {
                     Text = "~"
-                    UpperLeftCorner = GridConvert.upperLeftCornerOf { Row = i; Column = 0 }
+                    UpperLeftCorner = GridConvert.upperLeftCornerOf { Row = i * 1<mRow>; Column = 0<mColumn> }
                     Color = Colors.defaultColorscheme.DimForeground
                 }
             let linesWithNoTilde =
                 if bufferLines.Length = 0
                 then 1
                 else bufferLines.Length
-            [linesWithNoTilde..dimensions.Rows-1]
+            [linesWithNoTilde..(dimensions.Rows / 1<mRow> - 1)]
             |> List.map lineNotInBufferAsDrawingObject
 
         List.append (background :: bufferLines) rowsNotInBuffer
 
-    let private renderWindow (window : WindowView) =
-        let drawings = contentsAsDrawingObjects window.Dimensions window.Buffer
+    let private getCharacter (buffer : string list) cell =
+        buffer.[cell.Row/1<mRow>].[cell.Column/1<mColumn>].ToString()

```

Comment:
Thanks for the find! Apparently the file I had been testing with had at least some whitespace in all its "blank" lines.